_pkgdown.yml

```diff
@@ -1,4 +1,27 @@
-url: "usepa.github.io/EPATADA/"
+url: usepa.github.io/EPATADA/
 template:
   bootstrap: 5
   bootswatch: yeti
+  light-switch: true
+articles:
+- title: Articles
```

Comment:
@hillarymarler @wokenny13 
FYI - if the names change or if we add new ones, we should now change them here too.